apps/dav/lib/Server.php

```diff
@@ -177,7 +178,9 @@ public function __construct(IRequest $request, string $baseUri) {
 
 			$this->server->addPlugin(\OC::$server->get(\OCA\DAV\CalDAV\Trashbin\Plugin::class));
 			$this->server->addPlugin(new \OCA\DAV\CalDAV\WebcalCaching\Plugin($request));
-			$this->server->addPlugin(new \Sabre\CalDAV\Subscriptions\Plugin());
+			if (\OC::$server->getConfig()->getAppValue('calendar', 'allow_subscribe_link', 'yes') === 'yes') {
```

Comment:
```suggestion
			if (\OC::$server->getConfig()->getAppValue('dav', 'allow_subscribe_link', 'yes') === 'yes') {
```
Makes more sense in the `dav` app. Needs https://github.com/nextcloud/calendar/pull/4573 to be adjusted too.

Reply:
The calendar app is only a CalDAV client, settings in this app should only concern UI stuff. You can use subscriptions without the calendar app (for instance with DAVx5).

To make things clear from other dav stuff I suggest you rename `allow_subscribe_link` in `allow_calendar_subscriptions`.

Reply:
Thanks for clarification. Param name [changed](https://github.com/nextcloud/server/pull/34387/commits/6128163b8a56a7c3f13d7a4f56beb74d6d7622eb) to `dav.allow_calendar_link_subscriptions` to distinguish it from other subscription methods.